appveyor.yml

```diff
@@ -56,12 +56,13 @@ install:
         Where-Object pullRequestId -eq $env:APPVEYOR_PULL_REQUEST_NUMBER)[0].buildNumber) { `
           throw "There are newer queued builds for this pull request, failing early." }
 
-    # We don't need to build the example recipe.
-    - cmd: if exist recipes\example rmdir recipes\example /s /q
-    # Skip if we are not in a PR.
-    - cmd: if "%APPVEYOR_REPO_NAME%" == "conda-forge/staged-recipes" if not defined APPVEYOR_PULL_REQUEST_NUMBER rmdir /q /s recipes
-    # Make sure the `recipes` directory exists if we removed it.
-    - cmd: if not exist recipes mkdir recipes
```

Comment:
It appears we should have kept this line and the previous one. In theory, they should no longer be needed as the code below will remove everything that is in `master`.

However, in practice, the AppVeyor builds lag a bit on Travis build that does the conversion. As a result, we don't find anything other than the `example` in `recipes` most of the time as evidenced by this [build]( https://ci.appveyor.com/project/conda-forge/staged-recipes/build/1.0.6235/job/e404oxjdk395c0ts#L190 ) on `master`. While we could restore these two lines, the speed up given by them isn't huge on easy builds (e.g. pure Python) as we still waste time installing Miniconda and such. Though it will hurt on the longer running builds.

While we can add these back, it feels like a waste of time to me. Not that it isn't worthwhile to fast finish those builds on `master`, but that we can do a much better job. Already, there is a much better strategy ready to go in PR ( https://github.com/conda-forge/staged-recipes/pull/1142 ). That PR will avoid downloading Miniconda or doing any real setup at all. Not to mention it is well tested. So I'd recommend we just merge it.